_source/logzio_collections/_tracing-sources/python-otel-auto.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Sending traces from Python applications via auto instrumentation with OpenTelemetry
+logo:
+  logofile: python.svg
+  orientation: vertical
+data-source: Automatic Python instrumentation
+templates: ["network-device-filebeat"]
+contributors:
+  - nshishkin
+shipping-tags:
+  - new-instrumentation
+order: 1380
+---
+
+<!-- tabContainer:start -->
+<div class="branching-container">
+
+* [Overview](#overview)
+* [Local host](#local-host)
+{:.branching-tabs}
+
+<!-- tab:start -->
+<div id="overview">
+
+Deploy this integration to enable automatic instrumentation of your Python application using OpenTelemetry.
+
+### Architecture overview
+
+This integration includes:
+
+* Installing the OpenTelemetry Python instrumentation packages on your application host
+* Installing the OpenTelemetry collector with Logz.io exporter
+* Running your Python application in conjunction with the OpenTelemetry instrumentation
+
+On deployment, the Python instrumentation automatically captures spans from your application and forwards them to the collector, which exports the data to your Logz.io account.
+
+</div>
+<!-- tab:end -->
+
+
+<!-- tab:start -->
+<div id="local-host">
+
+
+### Setup auto-instrumentation for your locally hosted Python application and send traces to Logz.io
+
+**Before you begin, you'll need**:
+
+* a Python application without instrumentation
```

Comment:
Inconsistent formatting? 
In other places, the first word of the list is capitalized. 
I noticed this in another prerequisite section for a PR I just approved